src/python/grpcio/tests/unit/framework/foundation/_logging_pool_test.py

```diff
@@ -29,6 +29,8 @@
 
 """Tests for grpc.framework.foundation.logging_pool."""
 
+from __future__ import division
```

Comment:
Let's have the standard for the codebase be "presence of this import indicates that the module cannot function as intended without it".

Ideally the same standard as any other code element. :-)